nepalingo-web/src/lib/getGTranslate.tsx

```diff
@@ -0,0 +1,45 @@
+const GOOGLE_TRANSLATE_API_KEY = "AIzaSyCi239eredIAEEJkI8xzvGwW1GPS7B1vxQ";
+
+const languageCodes: { [key: string]: string } = {
+  maithili: "mai",
+  newari: "new",
+};
+
+export const getGTranslate = async (
+  language: string,
+  word: string,
+): Promise<{
+  language: string;
+  word: string;
+  meanings: Array<{
+    language: string;
+    meaningOriginal: string;
+    meaningEn: string;
+  }>;
+}> => {
+  const targetLanguage = languageCodes[language];
+  if (!targetLanguage) {
+    throw new Error(`Language code for ${language} not found`);
```

Comment:
```suggestion
    throw new Error(`Google translate language code for ${language} not found`);
```

Reply:
This change will make it more clear why the error is occurring and hints at what file to look at.